src/igvfd/types/file.py

```diff
@@ -550,8 +550,13 @@ def unique_keys(self, properties):
             'notSubmittable': True
         }
     )
-    def content_summary(self, request, dimension1, dimension2, content_type):
-        return f'{dimension1} by {dimension2} {content_type}'
+    def content_summary(self, request, dimension_x: str, dimension_y: list, content_type: str) -> str:
+        if len(dimension_y) > 1:
+            dimension_y_str = f'[{", ".join(dimension_y)}]'
```

Comment:
I am not sure I like having the square brackets around the "dimension y" values. Do you think it was too confusing if it looks like: "gene by cell, treatment transcriptome annotations"? 
Trying to think if there is any other way to represent it.